modules/minimum-required-permissions-ipi-azure.adoc

```diff
@@ -0,0 +1,279 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-account.adoc
+
+[id="minimum-required-azure-permissions_{context}"]
+= Required Azure permissions for installer-provisioned infrastructure
+
+When you assign `Contributor` and `User Access Administrator` roles to the service principal, you automatically grant all the required permissions.
+
+If you do not assign the `Contributor` and `User Access Administrator` roles to the service principal, the following permissions are required for creating an {product-title} cluster on Microsoft Azure.
```

Comment:
```suggestion
If your organization's security policies require a more restrictive set of permissions, you can create a custom role with the necessary permissions. The following permissions are required for creating an {product-title} cluster on Microsoft Azure.
```
Minor suggestion from Mike Pytlak so that users understand why they would want to use these permissions. Apply this to the UPI procedure as well please. Sorry for having you edit this line twice.

Reply:
Please update the UPI part also.

Reply:
Done